operator/pkg/images/env.go

```diff
@@ -4,12 +4,15 @@ import "github.com/stackrox/rox/pkg/env"
 
 // Environment variable settings for related image overrides.
 var (
-	Main          = env.RegisterSetting("RELATED_IMAGE_MAIN")
-	CentralDB     = env.RegisterSetting("RELATED_IMAGE_CENTRAL_DB")
-	Scanner       = env.RegisterSetting("RELATED_IMAGE_SCANNER")
-	ScannerSlim   = env.RegisterSetting("RELATED_IMAGE_SCANNER_SLIM")
-	ScannerDB     = env.RegisterSetting("RELATED_IMAGE_SCANNER_DB")
-	ScannerSlimDB = env.RegisterSetting("RELATED_IMAGE_SCANNER_DB_SLIM")
-	CollectorSlim = env.RegisterSetting("RELATED_IMAGE_COLLECTOR_SLIM")
-	CollectorFull = env.RegisterSetting("RELATED_IMAGE_COLLECTOR_FULL")
+	Main             = env.RegisterSetting("RELATED_IMAGE_MAIN")
+	CentralDB        = env.RegisterSetting("RELATED_IMAGE_CENTRAL_DB")
+	Scanner          = env.RegisterSetting("RELATED_IMAGE_SCANNER")
+	ScannerSlim      = env.RegisterSetting("RELATED_IMAGE_SCANNER_SLIM")
+	ScannerDB        = env.RegisterSetting("RELATED_IMAGE_SCANNER_DB")
+	ScannerSlimDB    = env.RegisterSetting("RELATED_IMAGE_SCANNER_DB_SLIM")
+	CollectorSlim    = env.RegisterSetting("RELATED_IMAGE_COLLECTOR_SLIM")
+	CollectorFull    = env.RegisterSetting("RELATED_IMAGE_COLLECTOR_FULL")
+	ScannerV4DB      = env.RegisterSetting("RELATED_IMAGE_SCANNER_V4_DB")
+	ScannerV4Indexer = env.RegisterSetting("RELATED_IMAGE_SCANNER_V4")
```

Comment:
Not sure yet if this is WIP or not, but could you elaborate on the meaning of this? In particular, why we split by service on the LHS and unify on the RHS?